cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -1996,7 +1996,9 @@ def test_obsm_values_warn_start_with_X(self, validator_with_adata):
         assert validator.warnings == [
             "WARNING: Dataframe 'var' only has 4 rows. Features SHOULD NOT be filtered from expression matrix.",
             "WARNING: Embedding key in 'adata.obsm' harmony does not start with X_ and thus will not be available in Explorer",
-            "WARNING: All embeddings have to be of 'numpy.ndarray' type, 'adata.obsm['harmony']' is <class 'pandas.core.frame.DataFrame'>').",
+        ]
+        assert validator.errors == [
+            "WARNING: All embeddings have to be of 'numpy.ndarray' type, 'adata.obsm['harmony']' is <class 'pandas.core.frame.DataFrame'>')."
```

Comment:
nit: we should be consistent and append errors with ERROR (blocks validation) rather than WARNING (doesn't)